frontend/lib/src/components/widgets/DateInput/DateInput.tsx

```diff
@@ -296,6 +296,10 @@ function DateInput({
               "::after": {
                 borderColor: colors.transparent,
               },
+              //Apply background color only when hovering over a date in the range in light theme
+              ...(hasLightBackgroundColor(theme) && $isHovered && $pseudoSelected && !$selected ? {
+                color: `${colors.secondaryBg} !important`,
+              } : {}),              
```

Comment:
I just tried not using the !important, and everything seems to work fine. Did you find a specific reason here that it is necessary?